README.md

```diff
@@ -24,26 +31,26 @@ This distribution comes with the following defaults:
 - Unlimited default limits for [configuration options](#trace-configuration) to
   support full-fidelity traces.
 
-If you're using the SignalFx Tracing Library for Ruby and want to migrate to the Splunk Distribution of OpenTelemetry Ruby, see [Migrate from the SignalFx Tracing Library for Ruby](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=ruby.migrate) in the official documentation.
+If you're using the SignalFx Tracing Library for Ruby and want to migrate to the Splunk Distribution of OpenTelemetry Ruby, see [Migrate from the SignalFx Tracing Library for Ruby](https://docs.splunk.com/observability/en/gdi/get-data-in/application/ruby/distro/troubleshooting/migrate-signalfx-ruby-agent-to-otel.html) in the official documentation.
 
 ## Requirements
 
 This distribution requires Ruby version 3.0 or higher.
 
 ## Get started
 
-For complete instructions on how to get started with the Splunk Distribution of OpenTelemetry Ruby, see [Instrument a Ruby application for Splunk Observability Cloud](https://quickdraw.splunk.com/redirect/?product=Observability&version=current&location=ruby.application) in the official documentation.
+For complete instructions on how to get started with the Splunk Distribution of OpenTelemetry Ruby, see [Instrument a Ruby application for Splunk Observability Cloud](https://docs.splunk.com/observability/en/gdi/get-data-in/application/ruby/distro/instrumentation/instrument-ruby-application.html) in the official documentation.
```

Comment:
I feel like the links that point to "old" splunk-distro documentation (everything with `/distro` should indicate in some way that this is a deprecated method/doc link?